README.md

```diff
@@ -259,11 +259,13 @@ A categorized community-driven collection of high-quality, awesome [LÖVE](http:
 * [Atom](https://atom.io/) - A hackable text editor for the 21st Century
 	* [Löve Atom](https://atom.io/packages/love-atom) - A lövely provider for Atom's autocomplete+ package
 	* [Löve IDE](https://atom.io/packages/love-ide) - This package auto-installs several utilities for writing Love2D games in Atom
+		* [Autocomplete GLSL](https://atom.io/packages/autocomplete-glsl) - Adds GLSL autocompletion in Atom
 		* [Autocomplete Löve](https://atom.io/packages/autocomplete-love) - Auto-complete and snippets for LÖVE
 		* [Hyperclick Löve](https://atom.io/packages/hyperclick-love) - A Hyperclick provider for LÖVE which shows the wiki
-	* [Löve Launcher](https://atom.io/packages/love-launcher) - Launch LÖVE 2D for the current project without having to leave Atom
-	* [language-lua](https://github.com/FireZenk/language-lua) - Lua language support in Atom
-	* [linter-luacheck](https://github.com/AtomLinter/linter-luacheck) - Lint Lua on the fly, using luacheck
+		* [language-glsl](https://atom.io/packages/language-glsl) - OpenGL Shading Language support in Atom
+		* [language-lua](https://atom.io/packages/language-lua) - Lua language support in Atom
+		* [linter-luaparse](https://atom.io/packages/linter-luaparse) - Lua syntax error checking in Atom

```

Comment:
Luacheck is probably a better linter though, as the repo page says "Unfortunately, luaparse only reports the first syntax error in a given file."